dev/doc/MERGING.md

```diff
@@ -70,7 +70,8 @@ To merge the PR proceed in the following way
 ```
 $ git checkout master
 $ git pull
-$ dev/tools/merge-pr XXXX
+$ git config commit.gpgsign true  # if GPG signing is not on by default
```

Comment:
You are not supposed to need this? What happens if it is not set?

Reply:
After creating my gpg key I still had an error message about my config user.signing data.
But now I don't have it anymore even in a fresh clone...
Maybe it was due to the gpg key being generated after cloning?

Reply:
I wouldn't think so. Was it an error message or just a warning?

Reply:
It sounds related to this bit in the script (which should only print stuff, not cause an error):
```bash
# Sanity check: user.signingkey
if [ -z "$(git config user.signingkey)" ]; then
  warning "git config user.signingkey is empty"
  warning "gpg will guess a key out of your git config user.* data"
fi
```
But without seeing the error it's hard to say.